packages/easy-testing/src/DataProvider/StaticFixtureFinder.php

```diff
@@ -8,29 +8,73 @@
 use Symfony\Component\Finder\Finder;
 use Symfony\Component\Finder\SplFileInfo;
 use Symplify\SmartFileSystem\SmartFileInfo;
+use Symplify\SymplifyKernel\Exception\ShouldNotHappenException;
 
 final class StaticFixtureFinder
 {
+    /**
+     * @var bool
+     */
+    public static $enableValidation = true;
+
     public static function yieldDirectory(string $directory, string $suffix = '*.php.inc'): Iterator
     {
         $fileInfos = self::findFilesInDirectory($directory, $suffix);
 
         foreach ($fileInfos as $fileInfo) {
             yield [new SmartFileInfo($fileInfo->getRealPath())];
         }
+
+        static::$enableValidation = true;
     }
```

Comment:
It's better to have standalone method, e.g. `yieldDirectoryExclusively()`

Reply:
implemented.